go.mod

```diff
@@ -294,14 +320,15 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
-	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-	github.com/btcsuite/btcutil => github.com/btcsuite/btcutil v1.0.2
+	github.com/Gravity-Bridge/Gravity-Bridge/module => github.com/umee-network/Gravity-Bridge/module v1.4.2-0.20220805205452-2445605a0377
 	github.com/bugsnag/panicwrap => github.com/bugsnag/panicwrap v1.3.4
```

Comment:
I think we can remove replace for `panicwrap`. @facundomedica you added it. Can you confirm? It seams no dependency is using it.

Reply:
LGTM, feels good to have less and less replace

Reply:
I removed the `keyring` and `btcutil` replace, but I was not sure about `panicwrap` - so my question was about `panicwrap`. Do you recall why did you add it?